src/rgw/rgw_crypt.cc

```diff
@@ -35,8 +35,6 @@ using namespace rgw;
 /**
  * Encryption in CTR mode. offset is used as IV for each block.
  */
-#warning "TODO: move this code to auth/Crypto for others to reuse."
```

Comment:
@amitkumar50 
In addition to my previous comment, it is ridiculous to purposefully create warnings for the TODO comments in the code.